tests/zfs-tests/tests/perf/regression/setup.ksh

```diff
@@ -0,0 +1,31 @@
+#!/bin/ksh
+
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2015 by Delphix. All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+[[ -z  $FIO ]] || \
+[[ -z  $FREE ]] || \
+[[ -z  $IOSTAT ]] || \
+[[ -z  $LSBLK ]] || \
+[[ -z  $MPSTAT ]] || \
+[[ -z  $VMSTAT ]] && \
+    log_fail "missing required commands"
```

Comment:
It would be nice to do this as 6 individual checks each with a unique error message listing the missing command.  This way when this fails you'll have an idea what command is missing.